content/blog/2016/2016-09-28-jenkins-world-2016-wrap-up-hub.adoc

```diff
@@ -0,0 +1,149 @@
+---
+layout: post
+title: "Jenkins World 2016 Wrap-up - Open Source Hub"
+tags:
+- event
+- jenkinsworld
+author: lnewman
+---
+
+NOTE: This is a guest post by link:https://github.com/bitwiseman[Liam Newman],
+Technical Evangelist at CloudBees.
+
+image:/images/conferences/Jenkins-World_125x125.png[Jenkins World 2016, role=right]
+
+As I mentioned in my
+link:/blog/2016/2016-09-27-jenkins-world-2016-wrap-up-scaling[previous] post
+link:https://www.cloudbees.com/jenkinsworld/home[Jenkins World] brings together
+Jenkins users of all scales.  It also bring together users from all skill levels, from
```

Comment:
"of all scales" reads weird to me, hard to describe since I get what you're alluding to.

Perhaps "Jenkins users from organizations of all sizes?"

> It also bring together users from all skill levels, from beginners to experts [..]

Should be:

> It also brought together users from all skill levels; from beginners to experts [..]